libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js

```diff
@@ -0,0 +1,51 @@
+(function () {
+    "use strict";
+
+    const exec = require("child_process").exec,
+        path = require("path");
+
+    let _domainManager;
+
+    function init(domainManager) {
+        _domainManager = domainManager;
+        if (!_domainManager.hasDomain("debugDomain")) {
+            _domainManager.registerDomain("debugDomain", {major: 0, minor: 1});
+        }
+
+        function handleWasm2Wast(projectId, src, callback) {
+            const sourcePath = path.join(process.cwd(), "projects", projectId);
+            const source = path.join(sourcePath, src);
+            const output = path.join(path.dirname(source), path.basename(src, ".wasm") + ".wast");
+
+            const wabtPath = path.join(process.cwd(), "tools", "wabt");
+            const wasm2wastBin = path.join(wabtPath, "bin", "wasm2wast");
+
+            const command = wasm2wastBin + " " + source + " -o " + output;
+
+            let child = exec(command, { cwd: sourcePath });
+
+            child.on("exit", () => {
+                callback(null);
+            });
+
+            child.on("error", (err) => {
+                callback(err);
+            });
+        }
+
+        _domainManager.registerCommand(
+            "debugDomain",
```

Comment:
debugDomain string seems to be used repeatly.
What about using string definision or variable ?

Reply:
Done.